frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/volume-snapshot.ts

```diff
@@ -1,14 +1,8 @@
-export const numberOfSnapshot = {
-  3: '3',
-  5: '5',
-  7: '7',
-};
-
 export const snapshotTypes = {
   Single: 'Single',
+  /* Other ways to create snapshot 
```

Comment:
Since you have removed all the code for snapshot other than single, it makes sense to remove these comments too. WDYT?